include/mrdox/Metadata/FieldType.hpp

```diff
@@ -31,7 +31,7 @@ union FieldFlags
 };
 
 // Info for field types.
-struct FieldTypeInfo
+struct FieldInfo
```

Comment:
What is a `FieldInfo`? Can we please add a doc comment to this struct...

Reply:
I think you are looking at an outdated version of the PR... `Field.hpp` has the new declaration & specifies that `FieldInfo` is for non-static data memebers